CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
+- default value of `false` for `c3.ad.isSlate` per Conviva Custom Ad Manager integration docs
```

Comment:
```suggestion
- Default value of `false` for `c3.ad.isSlate` for client side ad insertion per Conviva Custom Ad Manager integration docs
```
NIT to be in line with existing Changelog entries and narrow the effected use case.